guides/micronaut-microservices-distributed-tracing-jaeger-opentelemetry/metadata.json

```
@@ -0,0 +1,22 @@
+{
+  "title": "Use OpenTelemetry with Jaeger and the Micronaut Framework for Microservice Distributed Tracing",
+  "intro": "Use Jaeger distributed tracing to investigate the behaviour of your Micronaut applications.",
```

```suggestion
  "intro": "Use Jaeger distributed tracing to investigate the behavior of your Micronaut applications.",
```
Change to American spelling optional